spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'associations' do
+    it { is_expected.to have_many(:posts).with_foreign_key(:author_id) }
+    it { is_expected.to have_many(:likes) }
+    it { is_expected.to have_many(:comments) }
+  end
+
+  describe 'validations' do
+    subject do
+      User.new(name: 'Shahadat Hossain',
+               photo: 'https://photo.com', bio: 'A skilled design and installation.')
+    end
+
+    before { subject.save }
+
```

Comment:
- [ ] As the previous reviewer mentioned, you missed the test for the `recent_posts` method. Kindly add it